plugins/salesforce-einstein-api/CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## Instructions
+
+These steps will guide you through contributing to this project:
+
+- Fork the repo
+- Clone it and install dependencies
+
+      		git clone https://github.com/YOUR-USERNAME/typescript-library-starter
```

Comment:
Maybe change the contents of this file to actually match this repo, or just remove the file?